conversions/src/Data/Conversions.hs

```diff
@@ -146,6 +147,9 @@ instance Conversion Natural Word32 where
 instance Conversion Natural Word64 where
   convert = fromIntegral
 
+instance Conversion Integer a => Conversion Scientific a where
+  convert = fromInteger . convert
```

Comment:
I believe this should be the same as `Scientific.scientific (convert value) 0`

Reply:
@epicallan I'm cautiously watching "via" instances, where the source type is polymorphic. I'm not sure this is the right direction for us, but cannot right now refute it.

Reply:
> I'm cautiously watching "via" instances, where the source type is polymorphic.

Is this because of possible overlapping instance issues?

Reply:
Using this updated version is introducing overlapping instances errors in `vahana`. I am going to make an update where I have only a couple of common monomorphic to Scientific conversion instances.

Reply:
yes I feared that. I think we should provide an explicit `via` function for this.

And provide explicit from `Integer` to `Scientific`.

and if we ever had to construct from something that goes via `Integer` we can do: `convertVia @Integer value`. Can be followup.